### PR TITLE
[ADD] web_edit_mode addon

### PR DESCRIPTION
**Pull request by mharenz**
Add new module with that the user can change the read mode to edit mode and do not have to click the edit button every time he wants to change something in a form view.

### CODE REVIEW

File: web_edit_mode/models/res_users.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```
Comment (yweng8111) [inline web_edit_mode/models/res_users.py:1]: You have forgot the licence and copyright here.

**Comment by Tardo** [comment]
On runbot with this module can't edit any record... all time opens in "only-read mode" (except some type of fields like m2o).

**Comment by pedrobaeza** [comment]
For me this concept is very dangerous, as the edit mode is there for a reason (not a technical one, but readability + security one).

**Comment by mharenz** [comment]
@Tardo 
> 
> On runbot with this module can't edit any record... all time opens in "only-read mode" (except some type of fields like m2o).

You have to change it for your user as in configuration (README.rst) described:
At user settings the user can change the default behavior from read mode
to edit mode.

I will soon make the description clearer.

@pedrobaeza 
> 
> For me this concept is very dangerous, as the edit mode is there for a reason (not a technical one, but readability + security one).

We are running odoo (version 8) in a manufacturing company with 120 users since 4 years. This was one of the first improvements that our users wanted. Some users did change data by mistake, but this cases were only at beginning.

**Comment by pedrobaeza** [comment]
OK, this can host any opinion on this matter, so go ahead, but I don't share your PoV on the safety part and prefer to leave dual mode, plus extra of readability of the data on view mode, that I appreciate.

**Comment by Tardo** [comment]
@mharenz I see it, but some views doesn't work properly. Try it on invoice model in form view.

**Pull request closed without merging**